src/main/groovy/com/contrastsecurity/VerifyContrast.groovy

```diff
@@ -4,35 +4,34 @@ import com.contrastsecurity.exceptions.UnauthorizedException
 import com.contrastsecurity.http.RuleSeverity
 import com.contrastsecurity.http.ServerFilterForm
 import com.contrastsecurity.http.TraceFilterForm
-import com.contrastsecurity.models.Application
-import com.contrastsecurity.models.Applications
-import com.contrastsecurity.models.Servers
-import com.contrastsecurity.models.Trace
-import com.contrastsecurity.models.Traces
+import com.contrastsecurity.models.*
 import com.contrastsecurity.sdk.ContrastSDK
 import org.gradle.api.DefaultTask
 import org.gradle.api.GradleException
 import org.gradle.api.tasks.TaskAction
 
+import java.text.SimpleDateFormat
 
 class VerifyContrast extends DefaultTask {
 
-    Date startDate = new Date();
-
     @TaskAction
     def exec() {
         logger.debug('Checking for new vulnerabilities')
 
         ContrastSDK contrast = ContrastGradlePlugin.contrastSDK
         ContrastPluginExtension extension = project.contrastConfiguration
 
+        if (!ContrastGradlePlugin.appVersionQualifier) {
```

Comment:
You can take this out. No use initializing a new qualifier in the verify state. If it wasn't set in the install step, then there's no use checking with it in the verify step.

Reply:
Done, it is removed.